xgboost/notebooks/Online Prediction with XGBoost.ipynb

```diff
@@ -0,0 +1,627 @@
+    "| **2** | 28| 2 | 336951 | 7 | 12 | 2 | 11 |\n",
+    "\n",
+    "| **Person** | relationship | race | sex | capital-gain | capital-loss | hours-per-week | native-country || (Label) income-level|\n",
+    "|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:||:-:\n",
+    "| **1** | 3 | 2 | 1 | 0 | 0 | 40 | 38 || False (<=50K) |\n",
+    "| **2** | 0 | 4 | 1 | 0 | 0 | 40 | 38 || True (>50K) |\n",
+    "\n",
+    "\n",
+    "Creating a model version can take several minutes, check the status of your model version to see if it is available."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "NAME  DEPLOYMENT_URI                        STATE\r\n",
+      "v1    gs://xgboost-tutorial-notebook-test/  CREATING\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "! gcloud ml-engine versions list --model $MODEL_NAME"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "NAME  DEPLOYMENT_URI                        STATE\r\n",
+      "v1    gs://xgboost-tutorial-notebook-test/  READY\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "! gcloud ml-engine versions list --model $MODEL_NAME"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Test the model with an online prediction using the data of a person who makes <=50K.\n",
+    "\n",
+    "Note: If you see an error, the model from Part 4 may not be created yet as it takes several minutes for a new model version to be created."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[0.09139516949653625]\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "! gcloud ml-engine predict --model $MODEL_NAME --version $VERSION_NAME --json-instances less_than_50K.json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Test the model with an online prediction using the data of a person who makes >50K."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[0.4006653428077698]\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "! gcloud ml-engine predict --model $MODEL_NAME --version $VERSION_NAME --json-instances more_than_50K.json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use Python to make online predictions\n",
+    "We'll test the model with the entire test set and print out some of the results. Also, realise how the cells above return floats instead of booleans. Let's deal with that so the output type of the predictions match those of the test set labels.\n",
```

Comment:
I'd move your comment "realize how the cells ..." to a note above and say you'll fix it in the next step. Helps clarify why they got a float when maybe they weren't expecting that. Thoughts?